CONTRIBUTING.md

```diff
@@ -79,7 +79,13 @@ To build the Presto docs, see the [docs README](presto-docs/README.md).
 
 ### Building the Web UI
 
-The Presto Web UI is composed of several React components and is written in JSX and ES6. This source code is compiled and packaged into browser-compatible JavaScript, which is then checked in to the Presto source code (in the `dist` folder). You must have [Node.js](https://nodejs.org/en/download/) and [Yarn](https://yarnpkg.com/en/) installed to execute these commands. To update this folder after making changes, simply run:
+The Presto Web UI is composed of several React components and is written in JSX and ES6. This source code is compiled and packaged into browser-compatible JavaScript, which is then checked in to the Presto source code (in the `dist` folder). You must have [Node.js](https://nodejs.org/en/download/) and [Yarn](https://yarnpkg.com/en/) installed to execute these commands.
+
+Install Node.js from the link above and Yarn can be installed using brew:
+
+    brew install yarn
+
+To update this folder after making changes, simply run:
```

Comment:
```suggestion
To update this folder after making changes, run:
```
Nit: Technical documentation should not describe something as "simple" or "easy", because "simple" and "easy" are different for different users. For more context, see Gitlab documentation style guide topic [Building Trust](https://docs.gitlab.com/ee/development/documentation/styleguide/#building-trust), and [Don’t say “simply” in your documentation by Jim Fisher](https://blog.knowledgeowl.com/blog/posts/dont-say-simply-jim-fisher/).